CONTRIBUTING.md

```diff
@@ -345,6 +345,20 @@ By making a contribution to this project, I certify that:
     this project or the open source license(s) involved.
 ```
 
+### Typos
+
+We have a tool to check for typos. It's included in the `pre-commit`.
+I also runs in the CI.
```

Comment:
**Quibble** `s/I/It`

Reply:
The tool doesn't detect all typos haha, mostly words that don't exist. I'll fix that :)

Reply:
Just for your information, when you encounter this type of small fixes that may block the merging of a PR, you have this quick option:
![Capture d'écran 2024-04-18 093718](https://github.com/modularml/mojo/assets/12891691/69bcd2ba-4141-4be8-a054-dc78bf20e89e)
It allows you to change the code directly, thus fixing the typo. You can then click "Commit and sign". It will commit the change to the PR.
Obviously you should do it only when it's sure that the contributor won't mind that you've changed their code without prior warning, like with typos
That can save you some time, to avoid back and forth

Reply:
Awesome!  Thank you for pointing me at this suggestion tool!  I've used it in the past, but it seems the UI changed a bit, and I never bothered to find where it got moved to 😆 .

I'll definitely use it in the past to save you (and others) the back-and-forth over trivial fixes I can apply myself to speed things up.